RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -59,20 +86,52 @@
 kt4PFJetsForRho.GhostArea     = cms.double(0.005)
 
 from RecoHI.HiJetAlgos.hiFJRhoProducer import hiFJRhoProducer
+#from RecoHI.HiJetAlgos.hiPuRhoProducer_cfi import hiPuRhoProducer
+#from RecoHI.HiJetAlgos.hiFJRhoFlowModulationProducer_cfi import hiFJRhoFlowModulationProducer
+#from RecoHI.HiJetAlgos.hiPuRhoProducer import hiPuRhoProducer
+#from RecoHI.HiJetAlgos.hiFJRhoFlowModulationProducer import hiFJRhoFlowModulationProducer
+#from RecoHI.HiJetAlgos.hiPFCandCleaner_cfi import hiPFCandCleaner
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant